<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1298</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame_background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>512</width>
      <height>512</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_loadImage">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>250</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Incarca imaginea de fundal</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_changeBackgroundColor">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>60</y>
      <width>250</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Alege culoarea de fundal</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_chosenBackgroundColor">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>110</y>
      <width>250</width>
      <height>40</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QSlider" name="slider_threshold">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>160</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>256</number>
    </property>
    <property name="value">
     <number>128</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_histogram">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>250</y>
      <width>256</width>
      <height>256</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_threshold">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>0</y>
      <width>512</width>
      <height>512</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_thresholdValue">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>180</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Threshold Value:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_chosenThreshold">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>200</y>
      <width>250</width>
      <height>40</height>
     </rect>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="maxVisibleItems">
     <number>4</number>
    </property>
    <property name="insertPolicy">
     <enum>QComboBox::InsertAtTop</enum>
    </property>
    <item>
     <property name="text">
      <string>THRESH_BINARY</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>THRESH_BINARY_INV</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>THRESH_TRUNC</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>THRESH_TOZERO</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>THRESH_TOZERO_INV</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1298</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuT04">
    <property name="title">
     <string>T04</string>
    </property>
   </widget>
   <addaction name="menuT04"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
